src/main/java/org/opensearch/sdk/ExtensionSettings.java

```diff
@@ -30,6 +54,33 @@ public class ExtensionSettings {
     private String opensearchAddress;
     private String opensearchPort;
 
+    public static final Set<String> SECURITY_SETTINGS_KEYS = Set.of(
+        "path.home", // TODO Find the right place to put this setting
```

Comment:
I will create a follow-up issue. Similar to the opensearch `path.home` setting this would represent the directory where the extension is installed and shouldn't be within the security settings.